config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Webplatform::Application.routes.draw do
+  #get 'user_sessions/new'
+
+  #get 'user_sessions/create'
+
+  #get 'user_sessions/destroy'
+
+  resources :user_sessions
```

Comment:
there'll be alway one session so let's use singular resource here: `resource :user_session` and btw maybe we'll rename it to just session? but that's just an idea :)